src/signals/settings/components/PageHeader/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import PageHeader from 'components/PageHeader';
+
+const StyledPageHeader = styled(PageHeader)`
+  background-color: transparent;
+`;
+
+export const PageHeaderComponent = ({ title }) => (
```

Comment:
A `StyledPageHeader` can only takes a title and no other props?

Reply:
In this context, yes